src/Compilers/CSharp/Test/Semantic/Semantics/NameOfTests.cs

```diff
@@ -1168,5 +1168,66 @@ public void NameofTypeParameterInParameterInitializer()
 }";
             var compilation = CreateCompilationWithMscorlib45(source).VerifyDiagnostics();
         }
+
+        [Fact, WorkItem(10467, "https://github.com/dotnet/roslyn/issues/10467")]
+        public void NameofFixedBuffer()
```

Comment:
While you are in this file please add `[CompilerTrait(CompilerFeature.NameOf)]` to the class.